setup.py

```diff
@@ -227,7 +228,7 @@ def run(self):
 extras["modelcreation"] = deps_list("cookiecutter")
 
 extras["serving"] = deps_list("pydantic", "uvicorn", "fastapi", "starlette")
-extras["speech"] = deps_list("soundfile")
+extras["speech"] = deps_list("soundfile", "torchaudio")
```

Comment:
You're having issues in your tests because the `speech` category in the setup.py is contained in `testing`. The `speech` directive is installed in every test suite, so by adding `torchaudio` to it, you're installing `torch` + `torchaudio` to every suite.

This also explains why you have a timeout exceeded in your `run_tests_tf`: it runs all tests!